src/features/labels/links/PersonLink.tsx

```diff
@@ -56,47 +60,47 @@ export default function PersonLink({
     (admin) => admin.person.actor_id === person.actor_id,
   );
   const { hideUsernames } = useContext(ShareImageContext);
+  const currentUserHandle = useAppSelector(userHandleSelector);
```

Comment:
Because this isn't used for render, can you define this within the `onCommunityLinkLongPress` function?

```
const currentUserHandle = userHandleSelector(state);
```